http-server/src/main/java/io/airlift/http/server/HttpServer.java

```diff
@@ -97,8 +93,8 @@
     private final DelimitedRequestLog requestLog;
     private ConnectionStats httpConnectionStats;
     private ConnectionStats httpsConnectionStats;
-
-    private final Optional<ZonedDateTime> certificateExpiration;
+    private final ScheduledExecutorService scheduledExecutorService = new ScheduledThreadPoolExecutor(1);
```

Comment:
Nit: the canonical way is the `newScheduledThreadPool(1)` factory method from `Executors`